mobile/lib/repositories/etag.repository.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/entities/etag.entity.dart';
+import 'package:immich_mobile/interfaces/etag.interface.dart';
+import 'package:immich_mobile/providers/db.provider.dart';
+import 'package:immich_mobile/repositories/database.repository.dart';
+import 'package:isar/isar.dart';
+
+final etagRepositoryProvider =
```

Comment:
Should we invalidate this in `invalidateAllApiRepositoryProviders()` as well?

Reply:
no need, it's only accessing the local database